### ISSUE TITLE
Outdated dependencies

### ISSUE THREAD

**Issue by Edditoria** [stevelacy/gulp-cson#4]
Thanks for creating this npm. Is it possible to update this nice repo?

Here are information I inspect:

```shell
$ npm outdated # right after "npm i"
Package      Current  Wanted  Latest  Location
cson           3.0.2   3.0.2   7.5.0  gulp-cson
map-stream     0.1.0   0.1.0   0.0.7  gulp-cson
replace-ext    0.0.1   0.0.1   2.0.0  gulp-cson

$ npm run test
[...]
 × ERROR: --compilers is DEPRECATED and no longer supported.
          See https://git.io/vdcSr for migration information.
[...]
```

Coffeescript should be updated to v2 too.

Many thanks.

**Comment by stephenlacy** [comment]
Updated and published to 1.0.0 aa301ca348e0b9a4bd793e22c426e24357654d24

Cheers

**Comment by Edditoria** [comment]
What the... fast! 😄 
Thanks for the update!

Happy coding
